### PR TITLE
Automating pmp-revamp ui application

### PR DESCRIPTION
**Pull request by damodarguru**

### CODE REVIEW

File: uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/kernel/util/KeycloakUserManager.java [hunk 1]
```diff
@@ -86,8 +86,6 @@ public static void createUsers() {
 				moduleSpecificUser = needsToBeCreatedUser;
 			} else if (needsToBeCreatedUser.equals("nocert")) {
 				moduleSpecificUser = needsToBeCreatedUser;
```
Comment (anup-nehe) [inline uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/kernel/util/KeycloakUserManager.java:88]: have you added new user??

File: uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/pages/DashboardPage.java [hunk 2]
```diff
@@ -83,6 +83,7 @@ public class DashboardPage extends BasePage {
 	@FindBy(xpath = "//*[text()='Root of Trust Certificate']")
 	private WebElement RootOFTrustCertText;
 	
+	@FindBy(id = "rootCertificateList.uploadRootCaCertificate")
 	//@FindBy(id = "root_certificate_upload_btn")
```
Comment (anup-nehe) [inline uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/pages/DashboardPage.java:87]: delete the commented code if not require

File: uitest-pmprevamp/src/main/resources/config/Kernel.properties [hunk 3]
```diff
@@ -2,6 +2,13 @@
 #Make sure all resource apis are covered
 authenticationInternal=/v1/authmanager/authenticate/internal/useridPwd
 zoneMappingUrl=/v1/masterdata/zoneuser
+revampCreatePartnerUrl=/v1/partnermanager/partners
```
Comment (anup-nehe) [inline uitest-pmprevamp/src/main/resources/config/Kernel.properties:5]: don't push kernel. Properties

File: uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/kernel/util/KeycloakUserManager.java (hunk 1 above)
Comment (damodarguru) [reply uitest-pmprevamp/src/main/java/io/mosip/testrig/pmprevampui/kernel/util/KeycloakUserManager.java:88]: no, already created

**Comment by damodarguru** [comment]
Need to remove kernel properties.

**Pull request closed without merging**